test/unit/modules/isShopgateIosAppTest.js

```diff
@@ -0,0 +1,22 @@
+import chai from 'chai';
+import { isShopgateIosApp } from '../../../src/modules/isShopgateIosApp';
+
+const { expect } = chai;
+
+describe('isShopgateIosApp', () => {
+  it('should identify an IOS device', () => {
+    global.navigator = {
```

Comment:
Makes sense to cover a desktop user agent too.

Reply:
done